mriqc/workflows/diffusion/base.py

```diff
@@ -99,12 +99,14 @@ def dmri_qc_workflow(name="dwiMRIQC"):
     inputnode.iterables = [("in_file", dataset)]
 
     datalad_get = pe.Node(
-        DataladIdentityInterface(fields=["in_file"], dataset_path=config.execution.bids_dir),
+        DataladIdentityInterface(
+            fields=["in_file"], dataset_path=config.execution.bids_dir),
```

Comment:
There are still a large number of unrelated changes. Please `git checkout -p upstream/master` and revert any that are unrelated to this current PR.